.storybook/stories/DataFiltering/components/DataTable.test.tsx

```diff
@@ -0,0 +1,78 @@
+import '@testing-library/jest-dom';
```

Comment:
Do we need these tests? Just asking because `DataTable` should already be tested in the small, medium and large filter tests.

Reply:
I was flip-flopping between both ideas, happy to discuss :)